docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -426,6 +426,8 @@ const DebugPage: React.FunctionComponent<Props> = ({
       if (isAnnotatedService) {
         if (queries) {
           params.set('queries', queries);
+        } else {
+          params.delete('queries');
```

Comment:
I think `headers` might also have a similar issue (we clear it from the debug page, but the header is still sent)

Optionally, I think this can also be handled in this PR (or separately, whichever you prefer 😄 )

Reply:
Thanks, let me check and add it to this PR